packages/web-app-epub-reader/src/App.vue

```diff
@@ -31,9 +31,24 @@
 <script lang="ts">
 import { defineComponent, nextTick, PropType, ref, unref, watch } from 'vue'
 import { Resource } from '@ownclouders/web-client/src/helpers/resource/types'
-import { AppConfigObject, Key, useKeyboardActions } from '@ownclouders/web-pkg'
+import { AppConfigObject, Key, useKeyboardActions, useThemeStore } from '@ownclouders/web-pkg'
 import ePub, { Book, NavItem, Rendition } from 'epubjs'
 
+const DARK_THEME_CONFIG = {
+  html: {
+    '-webkit-filter': 'invert(1) hue-rotate(180deg)',
+    filter: 'invert(1) hue-rotate(180deg)'
+  },
+  img: {
+    '-webkit-filter': 'invert(1) hue-rotate(180deg)',
+    filter: 'invert(1) hue-rotate(180deg)'
+  }
+}
+
+const LIGHT_THEME_CONFIG = {
+  html: { background: 'white' }
```

Comment:
this is just a placeholder, we can update the theme to our own needs later.
but could also kick that.

FYI @kulmann